script.js

```diff
@@ -0,0 +1,43 @@
+const barcontainer = document.querySelector(".baricon");
+const toggle_button = document.querySelector("#nav-toggler");
+function barstogglers() {
+  barcontainer.classList.toggle("change");
+}
+
+function toggleNavMenu() {
+  const menu = document.querySelector(".nav-items");
+  barcontainer.addEventListener("click", barstogglers);
+  if (menu.style.display === "none") {
+    menu.style.display = "block";
+  } else if (menu.style.display == "flex") {
+    menu.style.display = "none";
+  } else {
+    menu.style.display = "none";
+  }
+}
+
+function addNavMenuToggleListener() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  toggleButton.addEventListener("click", toggleNavMenu);
+}
+
+function removeNavMenuToggleListener() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  toggleButton.removeEventListener("click", toggleNavMenu);
+}
+
+function checkScreenSize() {
+  const toggleButton = document.querySelector("#nav-toggler");
+  if (window.innerWidth <= 768) {
+    toggleButton.addEventListener("click", toggleNavMenu);
+  } else {
+    toggleButton.removeEventListener("click", toggleNavMenu);
+    const navMenu = document.querySelector(".nav-items");
+    if (navMenu.classList.contains("active")) {
+      navMenu.classList.remove("active");
+    }
+  }
+}
+
+window.addEventListener("load", checkScreenSize);
+window.addEventListener("resize", checkScreenSize);
```

Comment:
- [ ] Great job so far. However, there're few things you need to work on before your project could be approved.
    - Your design should look exactly like that of Figma design including all the texts on the mobile menu. All the mobile menu options (texts) are the sections on your portfolio. If an option is clicked, the page should redirect to that section on the page, and the mobile menu get disappeared. A this moment, nothings happens when any of the menu options is clicked. Kindly check the project requirements to re-confirm what's expected of you on this milestone.

- [ ] You should also try to make the mobile menu to be fixed when it appears, so that it would not move away when a user tries to scrolls up like I demostrated on the screen recording. Kindly fix that too.

### Your Design
![your-menu](https://user-images.githubusercontent.com/26597432/232776062-18148bb7-cc3a-48d5-9b7a-5231667bf0a7.png)

### Figma Design
![temp-menu](https://user-images.githubusercontent.com/26597432/232776091-1004f4c4-f288-4d3c-8108-adb438a84cb6.png)

### App behaviour at the moment
[app-behave.webm](https://user-images.githubusercontent.com/26597432/232776745-345e99a7-4dca-43f1-9c62-1171ab83646b.webm)